tests/unittest/_torch/test_executor_request_queue.py

```
@@ -316,141 +316,489 @@ def test_clear_canceled_req_ids(executor_queue):
+        1,
+        create_mock_request_with_py_schedule_params(attention_dp_rank=0,
+                                                    attention_dp_relax=True))
+    req2 = RequestQueueItem(
+        2,
+        create_mock_request_with_py_schedule_params(attention_dp_rank=1,
+                                                    attention_dp_relax=True))
+
+    scheduled_requests = []
+    unscheduled_requests = [req1, req2]
+
+    result = attention_dp_queue._schedule_attention_dp_requests(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)